engines/wintermute/ad/ad_scene.cpp

```diff
@@ -181,6 +190,12 @@ void AdScene::cleanup() {
 	}
 	_objects.clear();
 
+#ifdef ENABLE_WME3D
+	if (_sceneGeometry) {
```

Comment:
If shouldn't be necessary. (Either we consistently apply nullptr, and this if wouldn't work, or we don't and the if wouldn't have any effect on whether we'd be deleting a stale pointer)